src/lib/dapp/index.ts

```diff
@@ -288,87 +273,6 @@ export class DAppConnector {
     this.signers.push(...this.createSigners(session))
   }
 
-  private pingWithTimeout = async (
-    { topic }: EngineTypes.PingParams,
-    pingTimeoutMs: number = 1_000,
-  ) => {
-    return new Promise<void>(async (resolve, reject) => {
-      const timeout = setTimeout(() => {
-        reject(new Error(`Ping to ${topic} timed out after ${pingTimeoutMs}(ms)`))
-      }, pingTimeoutMs)
-
-      try {
-        resolve(await this.walletConnectClient?.ping({ topic }))
-      } catch (err) {
-        reject(err)
-      } finally {
-        clearTimeout(timeout)
-      }
-    })
-  }
-
-  private async pingWithRetry(topic: string, retries = 3): Promise<void> {
-    try {
-      await this.pingWithTimeout({ topic })
-    } catch (error) {
-      if (retries > 0) {
-        console.log(`Ping failed, ${retries} retries left. Retrying in 1 seconds...`)
-        await new Promise((resolve) => setTimeout(resolve, 1000))
-        await this.pingWithRetry(topic, retries - 1)
-      } else {
-        console.log(`Ping failed after ${retries} retries. Aborting...`)
-        throw error
-      }
-    }
-  }
-
-  private async checkPersistedState() {
```

Comment:
My review doesn't account for removing checkPersistedState or pingWithRetry. I don't think I'm qualified to review if these can be removed :)